flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -120,6 +123,28 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
     result.success("")
   }
 
+  private var didFetchAppStart = false
+
+  private fun fetchNativeAppStart(result: Result) {
+    val appStartTime = AppStartState.getInstance().getAppStartTime();
+    val isColdStart = AppStartState.getInstance().isColdStart();
+
+    if (appStartTime == null) {
+      result.error("1", "App start won't be sent due to missing appStartTime", null);
+    } else if (isColdStart == null) {
+      result.error("1", "App start won't be sent due to missing isColdStart", null);
+    } else {
+      val item = mapOf<String, Any?>(
+        "appStartTime" to appStartTime.getTime().toDouble(),
+        "isColdStart" to isColdStart,
+        "didFetchAppStart" to didFetchAppStart
+      )
+      result.success(item)
```

Comment:
The closures on `result` are called in all possible code paths though?

Reply:
I'm not sure, currently, all the methods are returning `result.success("")` so it's just a hunch.

Reply:
We now return `success(null)` and `success(nil)` and just print a waring if something went wrong. Returning an empty string would result in another error, as the deserializer would fail.